crates/worker-proto/proto/nemo/relay/worker/v1/plugin_worker.proto

```
@@ -44,6 +44,7 @@ message JsonValue {
 enum RegistrationSurface {
   REGISTRATION_SURFACE_UNSPECIFIED = 0;
   SUBSCRIBER = 1;
+  EVENT_METADATA_INJECTOR = 2;
```

Buf (1.72.0)

[error] Enum value name "EVENT_METADATA_INJECTOR" should be prefixed with "REGISTRATION_SURFACE_".

(ENUM_VALUE_PREFIX)